app/models/learning_object.rb

```diff
@@ -26,10 +26,16 @@ def link_concept(concept)
   end
 
   def seen? user_id
-    UserVisitedLoRelation.where(learning_object_id: self.id, user_id: user_id).count
+    self.user_to_lo_relations.select do |rel|
```

Comment:
Toto podla mna bude fungovat tak, ze to vyberie vsetky user_to_lo_relations daneho learning objectu, pre kazde musi vytvorit objekt a potom ich prechadza ako `Array#select`. Taketo filtrovanie by bolo vykonnejsie nechat na databaze ako predtym, len sa pozriet na indexy.